src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
+  font-family: avenir, sans-serif;
+  font-style: normal;
+  margin-bottom: 2%;
+  margin-top: 20px;
+  padding-left: 10%;
+}
+div.red_bar {
+  width: 70px;
+  margin-bottom: 2%;
+  border-top: 7px solid #df3143;
+  margin-left: 10%;
+}
+
+section#quick_id {
+  float: left;
+  color: white;
+  background-color: #2e4975;
+  width: 50%;
+  height: 300px;
+}
+section#contact_me {
+  float: right;
+  background-color: lightsteelblue;
+  width: 50%;
+  height: 300px;
+}
+
+table {
+  padding-left: 10%;
+  text-align: left;
+  font-size: 20px;
+  font-family: sans-serif;
+}
+table td {
+  padding-left: 20%;
+  font-weight: 200;
+}
+table th {
+  text-align: start;
+  text-combine-upright: none;
+  vertical-align: top;
+  font-weight: 700;
+}
+form {
+  padding-left: 5%;
+}
+form label {
+  display: none;
+}
+form input {
+  background-color: lightsteelblue;
+  color: white;
+  border: 2px hidden lightsteelblue;
+  border-radius: 4px;
+  border-bottom: 2px solid darkgrey;
+  padding: 12px 10px;
+  font-size: 16px;
+  margin-left: 5%;
+  margin-right: 20px;
+  width: 20%;
+}
+form input#subject {
+  display: block;
+  width: 50%;
+}
+form textarea {
+  width: 50%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 2px hidden lightsteelblue;
+  border-radius: 4px;
+  background-color: lightsteelblue;
+  font-size: 20px;
+  border-bottom: 2px solid darkgrey;
+  margin: 5%;
+  padding: 1px 10px;
+}
+input::placeholder {
+  font-style: normal;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 200;
+  font-size: 20px;
+}
+textarea::placeholder {
+  font-style: normal;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 200;
+  font-size: 20px;
+}
+input[type='submit' i] {
```

Comment:
Is this `i` a typo? And if you have used `id`, maybe just use `input#submit`

Reply:
Trying to avoid overuse id, just learn a way to select element.
for the 'i', it is not a typo, explain is down below https://www.w3.org/TR/selectors-4/#attribute-case
6.3. Case-sensitivity
By default case-sensitivity of attribute names and values in selectors depends on the document language.

To match attribute values case-insensitively regardless of document language rules, the attribute selector may include the identifier i before the closing bracket (]). When this flag is present, UAs must match the attribute’s value case-insensitively within the ASCII range.

Alternately, the attribute selector may include the identifier s before the closing bracket (]); in this case the UA must match the value case-sensitively (if possible) regardless of document language rules.

Like the rest of Selectors syntax, the i and s identifiers are case-insensitive within the ASCII range.

The following rule will style the frame attribute when it has a value of hsides, whether that value is represented as hsides, HSIDES, hSides, etc. even in an XML environment where attribute values are case-sensitive.
[frame=hsides i] { border-style: solid none; } 
The following rule will style lists with type="a" attributes differently than type="A" even though HTML defines the type attribute to be case-insensitive.
[type="a" s] { list-style: lower-alpha; }
[type="A" s] { list-style: upper-alpha; }
Note: Some document models normalize case-insensitive attribute values at parse time such that case-sensitive matching is impossible. Case-sensitive matching via s flags is only possible in systems that preserve the original case.

Reply:
"i" is not a typo,' i' stands for case-sensitivity flag in [6.3. Case-sensitivity](https://www.w3.org/TR/selectors-4/#attribute-case)

Reply:
hmmm, we normally don't use it for a reason: https://caniuse.com/#feat=css-case-insensitive
Please just use css class, so other people can just understand your code easily. Good to know this property though.